### ISSUE TITLE
segmentation fault  python train_ssd.py --datasets ~/data/VOC0712/VOC2007/ 

### ISSUE THREAD

**Issue by rinetd** [qfgaohao/pytorch-ssd#75]
运行环境： 
Darwin pytooldeMacBook-Pro.local 18.7.0 
Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64
opencv                    4.1.1
pytorch                   1.2.0

错误提示： 
/Users/pytool/miniconda3/envs/pytorch/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:82: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
/Users/pytool/miniconda3/envs/pytorch/lib/python3.7/site-packages/torch/nn/_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.
  warnings.warn(warning.format(ret))
zsh: segmentation fault  python train_ssd.py --datasets ~/data/VOC0712/VOC2007/ --validation_dataset

**Comment by rinetd** [comment]
i have solved it by Downgrade pytorch from pytorch-1.2.0 to pytorch-1.0
